src/note.ts

```diff
@@ -574,6 +582,8 @@ export abstract class Note extends Tickable {
       // Modifier spacing.
       modLeftPx,
       modRightPx,
+      totalLeftPx,
+      totalRightPx,
```

Comment:
@0xfe the yellow lines appear because `totalLeftPx` and `totalRightPx` before they were undefined causing a NaN in other part of the code I will also highlight.

Reply:
I think the problem is in `evaluate` -- it's the only place where there's a connection between `note.metrics.totalRightPx` and `formatterMetrics`.

What happens if you initialize `totalLeftPx` and `totalRightPx` here to `0`?

Alternatively, you might just be able to remove the references to `note.metrics.totalLeft/RightPx` from `evaluate()` (note: not the `tickContext` metrics properties, just the `note` ones.)

Reply:
They were 0 and they would also be there but overlapping with the other lines. Now they are, I believe, in the right position.

Reply:
Okay, I think we should remove it completely from here, because the names are not representative of the values. Instead, we should update `evaluate` to derive these values (just like you did here.)

Reply:
I copied the formula from:

https://github.com/0xfe/vexflow/blob/adda50f0d90212d5b8238f772d62b14a40288a17/src/tickcontext.ts#L245-L248

As you see above, they could be greater values

Reply:
This is `TickContext` which has different metrics from `Note`. Those operations are not run here.

Reply:
Okay, for the formatter, all that matters are the variables that are defined here -- we don't need `totalLeft/RightPx` here. We should remove them from here derive them in `evaluate` (they're only used for the debugging plots, AFACT.)